metadata-ingestion/src/datahub/ingestion/source/fivetran/fivetran_log_api.py

```diff
@@ -20,27 +20,44 @@ class FivetranLogAPI:
     def __init__(self, fivetran_log_config: FivetranLogConfig) -> None:
         self.fivetran_log_database: Optional[str] = None
         self.fivetran_log_config = fivetran_log_config
+        self.fivetran_log_query = FivetranLogQuery()
         self.engine = self._get_log_destination_engine()
 
     def _get_log_destination_engine(self) -> Any:
```

Comment:
it looks like this method is responsible for initializing engine, fivetran_log_database, and fivetran_log_query

can we make it return those in a tuple